lib/worldwide/config.rb

```diff
@@ -22,10 +22,17 @@ module Config
     OTHER_LOCALE_PATH_REGEX = /#{File::SEPARATOR}(?<locale>[\w-]+)\.(?:yml|rb)/
     REGIONS_LOCALE_PATH_REGEX = /#{File::SEPARATOR}(?<locale>[\w-]+)\.yml/
 
+    # Protects against multiple calls to configure_i18n with the same I18n::Config object
+    @previously_configured_i18ns = []
```

Comment:
I originally used a set instead of an array, then learned that sets have surprising behaviour for inclusion by object identity:
```ruby
(byebug) Worldwide::Config.previously_configured_i18ns.class.name
"Set"
(byebug) Worldwide::Config.previously_configured_i18ns.first === i18n_config
true
(byebug) Worldwide::Config.previously_configured_i18ns.include?(i18n_config)
false
(byebug) Worldwide::Config.previously_configured_i18ns.first.object_id
64480
(byebug) i18n_config.object_id
64480
```
I believe we're supposed to call [Set#compare_by_identity](https://docs.ruby-lang.org/en/master/Set.html#method-i-compare_by_identity), but there is no guaranteed support on all ruby flavours.

An array is just fine.